source/includes/php/errors.md

```diff
@@ -0,0 +1,58 @@
+# Errors
+
+> Catch an Error from the library
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+try {
+  $product = \Shoprunback\Elements\Product::retrieve('a wrong ID');
+} catch (\Shoprunback\Error\Error $e) {
+  // Here you can manage the error
+}
+```
+
+To prevent errors to crash your code, you must catch the exceptions.
+
+All the errors the library can return inherit from the class **\Shoprunback\Error\Error**. ->
+
+#### Parameters
+
+Parameter | Type | Description | Specific to
+-|-|-|-
+**message** | **String** | The mandatory message for an Exception | All
+**response** | **RestResponse** | The response from the server explaining why the call failed | **RestClientError**
+
+## Error types
+
+> Use a RestClientError
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('a wrong token');
+
+try {
+  $product = \Shoprunback\Elements\Product::retrieve('a wrong ID');
+} catch (\Shoprunback\Error\Error $e) {
+  if (is_a($e, \Shoprunback\Error\RestClientError)) {
+    $restResponse = $e->response;
+    // Then, see how to use a RestResponse
+  }
+}
+```
+
+Class name | Event | Special
+-|-|-
+**ElementCannotBeCreated** | Happens when the code tries to create an element that hasn't a POST endpoint (ex: Account)
```

Comment:
It's not the code, it's the user.

No need to say Happens when...

Just say what it is. ElementCannotBeCreated = When the element doesn't have a POST endpoint.

etc